rfc/11/README.md

```diff
@@ -20,7 +20,7 @@ Commands:
 
     bootstrap   bootstrap the cluster
     drive-state post drive event to Motr
-    fetch-fids  Fetch the fids for motr services (m0d and s3server)
+    fetch-fids  Fetch the fids for motr services (m0d and rgw)
```

Comment:
Same here, let's avoid mentioning rgw explicitly, better mention it as `motr clients, e.g. cortx-rgw, cortx-s3server, etc.`